<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>MainWindow</class>
 <widget class="QMainWindow" name="MainWindow">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>690</width>
    <height>421</height>
   </rect>
  </property>
  <property name="baseSize">
   <size>
    <width>1960</width>
    <height>1080</height>
   </size>
  </property>
  <property name="windowTitle">
   <string>MainWindow</string>
  </property>
  <widget class="QWidget" name="centralwidget">
   <layout class="QHBoxLayout" name="horizontalLayout">
    <item>
     <layout class="QGridLayout" name="gridLayout">
      <item row="0" column="0">
       <widget class="QLabel" name="file_path">
        <property name="text">
         <string>文件路径</string>
        </property>
       </widget>
      </item>
      <item row="0" column="3" rowspan="2">
       <widget class="QLabel" name="run_result">
        <property name="font">
         <font>
          <pointsize>15</pointsize>
         </font>
        </property>
        <property name="text">
         <string>运行结果</string>
        </property>
        <property name="alignment">
         <set>Qt::AlignCenter</set>
        </property>
       </widget>
      </item>
      <item row="2" column="0" colspan="3">
       <widget class="QListView" name="history_blog">
        <property name="font">
         <font>
          <pointsize>24</pointsize>
         </font>
        </property>
       </widget>
      </item>
      <item row="1" column="1">
       <widget class="QLineEdit" name="name_edit"/>
      </item>
      <item row="0" column="1">
       <widget class="QLabel" name="file_path_show">
        <property name="text">
         <string>   请选择</string>
        </property>
       </widget>
      </item>
      <item row="2" column="3">
       <widget class="QTextBrowser" name="run_result_show">
        <property name="font">
         <font>
          <pointsize>15</pointsize>
         </font>
        </property>
       </widget>
      </item>
      <item row="0" column="2">
       <widget class="QPushButton" name="choose_path">
        <property name="text">
         <string>选择路径</string>
        </property>
       </widget>
      </item>
      <item row="1" column="0">
       <widget class="QLabel" name="compile_name">
        <property name="text">
         <string>编译名称</string>
        </property>
       </widget>
      </item>
      <item row="1" column="2">
       <widget class="QPushButton" name="choose_name">
        <property name="text">
         <string>开始编译</string>
        </property>
       </widget>
      </item>
     </layout>
    </item>
   </layout>
  </widget>
  <widget class="QMenuBar" name="menubar">
   <property name="geometry">
    <rect>
     <x>0</x>
     <y>0</y>
     <width>690</width>
     <height>17</height>
    </rect>
   </property>
  </widget>
  <widget class="QStatusBar" name="statusbar"/>
 </widget>
 <resources/>
 <connections/>
</ui>
